.codepipeline/docker/cmfive_dev_tools.sh

```diff
@@ -22,7 +22,7 @@ cat <<EOF > /etc/php$PHP_VERSION/conf.d/50_xdebug.ini
 zend_extension=xdebug.so
 xdebug.mode=debug
 xdebug.start_with_request=yes
-xdebug.client_host=host.docker.internal
+xdebug.discover_client_host=true
```

Comment:
This does not work on my machine but reverting to specifying client_host still works. Since we're specifying the host in the docker-compose we should leave it locked to host.docker.internal

Reply:
Thx for testing it Adam. (Is advised as Xdebug3 'correct' approach, but no use if it does not work)